03.asynchronous/callback/callback.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+db.run(
+  "CREATE TABLE books(id INTEGER PRIMARY KEY AUTOINCREMENT,title TEXT NOT NULL)",
+  () => {
+    console.log("Booksテーブルを作成しました");
+    db.run("INSERT INTO books(title) VALUES(?)", "javascriptの本", () => {
```

Comment:
javascript ではなく JavaScript です。https://bootcamp.fjord.jp/practices/165#%E5%90%8D%E5%89%8D%E9%87%8D%E8%A6%81

```suggestion
    db.run("INSERT INTO books(title) VALUES(?)", "JavaScriptの本", () => {
```

Reply:
修正いたします！参考もありがとうございます！